Package.swift

```diff
@@ -11,6 +11,12 @@ import PackageDescription
 
 let package = Package(
     name: "Relay",
+    platforms: [
+        .macOS(.v10_11),
+        .iOS(.v8),
+        .tvOS(.v9),
+        .watchOS(.v2),
+    ],
```

Comment:
These values should match the ones in the Xcode project file (macOS should be `.v10_10`)

Reply:
good 👁